mev-boost-rs/src/relay_mux.rs

```diff
@@ -124,18 +126,40 @@ impl BlindedBlockProvider for RelayMux {
         &self,
         bid_request: &BidRequest,
     ) -> Result<SignedBuilderBid, BlindedBlockProviderError> {
-        let responses = stream::iter(self.relays.iter().cloned())
-            .map(|relay| async move { relay.fetch_best_bid(bid_request).await })
+        let try_responses = stream::iter(self.relays.iter().cloned())
```

Comment:
another nitpick
```suggestion
        let responses = stream::iter(self.relays.iter().cloned())
```

and then fixing the symbol down below